kvs-key-value-pairs/kvs-key-value-pairs.js

```diff
@@ -0,0 +1,24 @@
+import cf from 'cloudfront';
+​
+// Remember to associate the KVS with your function before referencing KVS in your code.
+// https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/kvs-with-functions-associate.html
+​
+async function handler(event) {
+    const request = event.request;
+    // Use the first segment of the pathname as key
+    // For example http(s)://domain/<key>/something/else
+    const pathSegments = request.uri.split('/')
+    const key = pathSegments[1]
+    try {
+        // Replace the first path of the pathname with the value of the key
+        // For example http(s)://domain/<value>/something/else
+        pathSegments[1] = await kvsHandle.get(key);
```

Comment:
You're not defining `kvsHandle` anywhere so this function would get an error. You need something like this: 
`const kvsHandle = cf.kvs();` outside the function handler.